src/SourceBuild/content/eng/Versions.props

```
@@ -32,5 +32,6 @@
     <MicrosoftExtensionsLoggingVersion>9.0.0-preview.2.24128.5</MicrosoftExtensionsLoggingVersion>
     <!-- command-line-api -->
     <SystemCommandLineVersion>2.0.0-beta4.24126.1</SystemCommandLineVersion>
+    <Microsoft_Extensions_FileSystemGlobbingVersion>7.0.0</Microsoft_Extensions_FileSystemGlobbingVersion>
```

For this version to get automatically updated eventually, this needs to follow the naming convention that Maestro understands:

```suggestion
    <MicrosoftExtensionsFileSystemGlobbingVersion>7.0.0</MicrosoftExtensionsFileSystemGlobbingVersion>
```